komga-webui/src/locales/en.json

```diff
@@ -557,32 +557,32 @@
     }
   },
   "error_codes": {
-    "ERR_1000": "File could not be accessed during analysis",
-    "ERR_1001": "Media type is not supported",
+    "ERR_1000": "Could not access file during analysis",
+    "ERR_1001": "Unsupported media type",
     "ERR_1002": "Encrypted RAR archives are not supported",
     "ERR_1003": "Solid RAR archives are not supported",
     "ERR_1004": "Multi-Volume RAR archives are not supported",
     "ERR_1005": "Unknown error while analyzing book",
-    "ERR_1006": "Book does not contain any page",
-    "ERR_1007": "Some entries could not be analyzed",
-    "ERR_1008": "Unknown error while getting book's entries",
+    "ERR_1006": "There are no pages in the book",
+    "ERR_1007": "Could not analyze some pages",
+    "ERR_1008": "Could not fetch the book's entries, for unknown reason",
     "ERR_1009": "A read list with that name already exists",
-    "ERR_1010": "No books were matched within the read list request",
-    "ERR_1011": "No unique match for series",
-    "ERR_1012": "No match for series",
-    "ERR_1013": "No unique match for book number within series",
+    "ERR_1010": "Could not find any books from read list request",
+    "ERR_1011": "Could not find that specific book in the series",
+    "ERR_1012": "Could not find anything like that in the series",
+    "ERR_1013": "Could not find this book number in the series",
     "ERR_1014": "No match for book number within series",
-    "ERR_1015": "Error while deserializing ComicRack ReadingList",
-    "ERR_1016": "Directory not accessible or not a directory",
-    "ERR_1017": "Cannot scan folder that is part of an existing library",
+    "ERR_1015": "Could not deserialize ComicRack ReadingList",
+    "ERR_1016": "Inaccessible or non-existing directory",
+    "ERR_1017": "Cannot scan a folder in an existing library",
```

Comment:
This seems weird, it doesn't clearly explain that the folder cannot be part of the existing library.